decode.go

```diff
@@ -570,10 +579,10 @@ type decoder struct {
 }
 
 // value decodes CBOR data item into the value pointed to by v.
-// If CBOR data item is invalid, error is returned and offset isn't changed.
-// If CBOR data item is valid but fails to be decode into v for other reasons,
+// If CBOR data item fails to be decode into v,
```

Comment:
Nice work!  Great to have 100% coverage in stream_test.go!

Didn't spot any problems in the code, but a comment has grammar error "fails to be decode into v".